src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -164,13 +182,19 @@ def __init__(
         # This flag records whether the simulation has been run
         self._has_run = False
 
+        # Should the tracker update particle energies after every time step to
+        # reflect stopping?
+        self._do_stopping = False
+
         # Raise a ValueError if a synchronized dt is required by termination condition or save routine but one is
         # not given. This is only the case if an array with differing entries is specified for dt
         if self._require_synchronized_time and not self._is_synchronized_time_step:
             raise ValueError(
                 "Please specify a synchronized time step to use the simulation with this configuration!"
             )
 
+        # Ensure that the grids have defined the necessary quantities or at least zeroed
+        self._required_quantities = self._REQUIRED_QUANTITIES.copy()
```

Comment:
I don't understand what's happening here? is _REQUIRED_QUANTITIES a Tracker property?

EDIT: Oh, I see, it's just defined below! Can you move it up, maybe above init, so one finds it before getting to this point when reading the code?